azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java

```diff
@@ -40,4 +40,15 @@ protected void doExecute() throws MojoExecutionException, MojoFailureException {
         // Update deployment, show url
         getLog().info(app.properties().url());
     }
+
+    @Override
+    protected void updateTelemetry(SpringConfiguration springConfiguration) {
```

Comment:
don't Override updateTelemetry, but a new method traceAuth and traceUsage

Reply:
rename to `traceTelemetry`